processor/ratelimitprocessor/metadata.yaml

```diff
@@ -29,6 +29,13 @@ telemetry:
         value_type: double
         monotonic: true
         bucket_boundaries: [ 0.0001, 0.0005, 0.001, 0.005, 0.01, 0.025, 0.05, 0.1, 0.5, 1.0, 5.0, 10.0 ]
+    ratelimit.request_size:
```

Comment:
Histogram is a wrong choice here, let's use a counter similar to how measure bytes in other components. We could do sum aggregation on top of it to get throughout per second etc.

Example - https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/ee30200e7499ce220793f87850f9f98b4bb4b83b/exporter/kafkaexporter/internal/metadata/generated_telemetry.go#L33

Reply:
Aah, so we want the total instead? Got it, I thought we wanted an idea of the size of the requests we were getting

Reply:
I take that back, Thinking this through - given we already have the throttled request count per tenant, Lets stick with histogram as we can calculate the distribution clearly not just the volume.